Exercise4/solution.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Loop over json data
+ * @param string $data The json string.
+ * @author aniket
+ */
+function convert($data){
+
+    $names = [];
+    $ages = [];
+    $city = [];
+    foreach($data["players"] as $player){
+        array_push($names,$player["name"]);
+        array_push($ages,$player["age"]);
+        array_push($city,$player["address"]);
+    }
+
+    echo "All names ,age and city into arrays each:\n";
+    print_r($names);
+    print_r($ages);
+    print_r($city);
+
+    echo "All unique names:\n";
+    $names = array_unique($names);
+    print_r($names);
+
+    echo "The name of Persons with max age:\n";
+    $maxValue = max($ages);
+    $maxAgesPersons = [];
+    foreach($data["players"] as $player){
+        if($player["age"] == $maxValue){
```

Comment:
This can be done in the previous if block

Reply:
but for that I will need maximum age as well from before